dev-docs/FAQ.adoc

```diff
@@ -0,0 +1,11 @@
+= Frequently Asked Questions
+:toc: left
+
+
+== FAQ
+
+This document provides a quick reference for frequently questioned concepts and features in Credentials.
+
+=== What do I do with NOTICE.txt if I don't have one to satisfy Gradle?
+
+You create a empty file, similar to the `/solr/licenses/antlr4-runtime-NOTICE.txt`
```

Comment:
[0] Not sure if you're looking for grammar edits at this point, but I'll leave them and you can choose to ignore if you want.

 "a empty file" -> "an empty file".

Reply:
Thanks...    BTW, in the future, if you want to edit my PR's, please do!  I also appreciate the detailed comments you leave me!

Reply:
Fair enough!  I never think to just fix this stuff myself, but it's easy enough and I don't mind.  Thanks for the invite.